app/locales/en.json

```diff
@@ -120,12 +120,15 @@
     "news_subtitle": "Read about the latest COVID updates from your health authority and in general.",
     "news_title": "Latest news",
     "no_data": "No data",
+    "privacy_policy": "Privacy Policy",
     "push_at_risk_message": "You have crossed paths with a COVID-19 patient",
     "push_at_risk_title": "You may be at risk",
     "see_exposure_history": "See exposure history",
     "settings_title": "Dashboard",
     "skip_this_step": "Skip this step",
-    "privacy_policy": "Privacy Policy"
+    "team": "Team",
```

Comment:
oh dammit. @alpita-masurkar can you make another PR to remove team, team_para and terms_of_use from en.json.. they are not in the app anymore